src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/web/VersionPageTests.kt

```diff
@@ -85,7 +93,7 @@ class VersionPageTests : IntegrationTest()
         Assertions.assertThat(result.statusCode).isEqualTo(200)
     }
 
-    private fun getAnyReportPageUrl(): Pair<String, String>
+    private fun getAnyReportPageUrl(latest: Boolean): Pair<String, String>
```

Comment:
I guess you could have given this new argument a default value of `false` to avoid having to change the existing tests